### ISSUE TITLE
FontFamily does not seem to work for spans on Android

### ISSUE THREAD

**Issue by stonecourier** [NativeScript/NativeScript#1131]
Setting via xml or code does not work when using FontAwesome
If you just set text property and font family it does work.

https://twitter.com/vladimir_enchev/status/660034284791681024
http://pastie.org/10517753#3

<pre>
 &lt;Label&gt;
          &lt;Label.formattedText&gt;
            &lt;FormattedString&gt;
              &lt;FormattedString.spans&gt;
                &lt;Span text="&#xf17b;" fontFamily="FontAwesome" /&gt;
                &lt;Span text="NativeScript"  /&gt;
              &lt;/FormattedString.spans&gt;
            &lt;/FormattedString&gt;
          &lt;/Label.formattedText&gt;
        &lt;/Label&gt;
</pre>

**Comment by enchev** [comment]
Hey @stonecourier,

Here is a temporary workaround:
```XML
<Page>
  <Label style="font-family: FontAwesome;">
    <Label.formattedText>
      <FormattedString>
        <FormattedString.spans>
          <Span ios:text="&#xf179;" android:text="&#xf17b;" />
          <Span text="NativeScript"  />
        </FormattedString.spans>
      </FormattedString>
    </Label.formattedText>
  </Label>
</Page>
```

**Comment by stonecourier** [comment]
Thanks, I got it working so I can now build several user controls based on FontAwesome http://stonecourier.blogspot.com.au/2015/11/using-font-awesome-to-build-your-own.html

**Comment by amjd** [comment]
@enchev Can you please explain me why the unicode value for the icon is different for iOS and Android? Also, how do we find these values?

**Comment by stonecourier** [comment]
Because on IOS, he is making it display the apple logo and on android the android logo. Values can be found here https://fortawesome.github.io/Font-Awesome/cheatsheet/

**Comment by amjd** [comment]
@stonecourier Thanks for the clarification! I should have spotted that. :grin:

**Comment by nsndeck** [comment]
Just a quick follow up - the original issue has been fixed with commit #1162.
